src/SeqCli/Templates/Export/TemplateSetExporter.cs

```diff
@@ -78,6 +79,13 @@ public async Task ExportTemplateSet()
             () => _connection.RetentionPolicies.ListAsync(),
             retentionPolicy => retentionPolicy.Id.Replace("retentionpolicy-", ""),
             templateValueMap);
+        
+        await ExportTemplates<ExpressionIndexEntity>(
+            id => _connection.ExpressionIndexes.FindAsync(id),
+            () => _connection.ExpressionIndexes.ListAsync(),
+            expressionIndex => expressionIndex.Id.Replace("expressionindex-", ""),
```

Comment:
`=> expressionIndex.Expression.All(char.IsLetterOrDigit) ? expressionIndex.Expression : expressionIndex.Id.Replace("expressionindex-", "")` might improve file naming for trivial indexes?